src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+
+type Props = {
+  error: string;
+  setError: (error: string) => void;
+};
+
+export const Notification: React.FC<Props> = (
+  { error, setError },
+) => {
+  useEffect(() => {
+    const timeout = setTimeout(() => setError(''), 3000);
+
+    return () => clearTimeout(timeout);
+  }, [error]);
+
+  return (
+    <div className="notification is-danger is-light has-text-weight-normal">
+      <button
+        type="button"
+        className="delete"
+        aria-label="close notification"
+        onClick={() => setError('')}
```

Comment:
You have duplicates of one callback ()=>setError(''). You can move them to separate function

Reply:
Thx 😊 Thats just a lifting state function, but took your advice and  did some changes